### PR TITLE
replace timezone with systemd-timesyncd

### PR DESCRIPTION
**Pull request by troy2914**
which does both the timezone and timesyncd.conf

### CODE REVIEW

File: group_vars/all/nac2022.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+timezone: America/Los_Angeles
```
Comment (nickygerritsen) [inline group_vars/all/nac2022.yml:2]: Does this now only overwrite the wf one because it gets loaded later? @vmcj what is a better way to do that?

Comment (vmcj) [reply group_vars/all/nac2022.yml:2]: I think setting this in the hosts file on the group (probably group[all] would be a better place. We already have that file being gitignored.

**Comment by vmcj** [comment]
The correct way would be to use a requirements file instead of a git submodule if this exists on ansible galaxy. Otherwise a `loosely based on` with citation would also be better IMO.

File: group_vars/all/nac2022.yml (hunk 1 above)
Comment (troy2914) [reply group_vars/all/nac2022.yml:2]: [WARNING]: Skipping key (timezone) in group (group[all]) as it is not a mapping, it is a <class 'ansible.parsing.yaml.objects.AnsibleUnicode'>
please post yaml :)

**Comment by troy2914** [comment]
> requirements

would that be a 
ansible-galaxy role install stuvusIT.systemd-timesyncd
?

**Comment by troy2914** [comment]
the actual command i used was:
$ ansible-galaxy install --roles-path . stuvusIT.systemd-timesyncd

and updated the role name in lastminute2022.yml

File: group_vars/all/nac2022.yml (hunk 1 above)
Comment (troy2914) [reply group_vars/all/nac2022.yml:2]: thinking this in hosts.yml: 
```
all:
  vars:
    timezone: America/New_York
```

Comment (troy2914) [reply group_vars/all/nac2022.yml]: this was pushed.

**Comment by vmcj** [comment]
I meant this:
https://galaxy.ansible.com/docs/using/installing.html#installing-multiple-roles-from-a-file

so for us that would be:
```
# from galaxy
- src: stuvusIT.systemd-timesyncd
```
as requirements.yml

and than on location (when there is internet run: ansible-galaxy install -r requirements.yml,

**Review by vmcj — APPROVED** [review]

**Pull request merged**